crates/network-scanner/src/ping.rs

```diff
@@ -19,58 +85,16 @@ pub fn blocking_ping(ip: Ipv4Addr) -> anyhow::Result<()> {
 
     let addr = SocketAddr::new(ip.into(), 0);
 
-    let (packet, verifier) = create_echo_request()?;
+    let (packet, _) = create_echo_request()?;
 
     socket
         .send_to(&packet.to_bytes(true), &addr.into())
         .with_context(|| format!("failed to send packet to {}", ip))?;
 
     let mut buffer = [MaybeUninit::uninit(); icmp_v4::ICMPV4_MTU];
-    let (size, _) = socket
+    let _ = socket
         .recv_from(&mut buffer)
         .with_context(|| format!("failed to receive packet from {}", ip))?;
 
-    // SAFETY: `recv_from` returns the number of bytes written into the buffer, so the `size` first
-    // elements are in an initialized state.
-    let filled = unsafe { assume_init(&buffer[..size]) };
-
-    let packet = icmp_v4::Icmpv4Packet::parse(filled).context("cannot parse icmp packet")?;
-
-    if let icmp_v4::Icmpv4Message::EchoReply { payload, .. } = packet.message {
```

Comment:
don't care about what the response is, we only care about there's a computer on the other end.

Reply:
Fair enough!